pegasus/tools/visualization.py

```diff
@@ -136,18 +143,22 @@ def calc_force_directed_layout(
     TODO: Typing
     """
     G = construct_graph(W)
+    try:
+        import forceatlas2 as fa2
+    except PackageNotFoundError:
+        print("Please install module forceatlas2")
+        exit()
```

Comment:
Not needed because this is within the `except` block.

Reply:
if we catch the exception without doing anything than printing, the script would continue its execution..

Reply:
Similarly as above, add "Exception:" at the beginning of the print statement in the except block.